velox/functions/prestosql/registration/ArrayFunctionsRegistration.cpp

```diff
@@ -52,6 +52,12 @@ inline void registerArrayCombinationsFunctions(const std::string& prefix) {
       int64_t>({prefix + "combinations"});
 }
 
+template <typename T>
+inline void registerArrayCumSumFunction(const std::string& prefix) {
+  registerFunction<ParameterBinder<ArrayCumSumFunction, T>, Array<T>, Array<T>>(
```

Comment:
Just checking : Does array_cum_sum work with Decimals ?

Reply:
It does, there was a decimal test added for the earlier implementation. With the current simple function, registering decimals in the signature would require more changes. The other array functions implemented as simple functions do not have decimal signatures as well, although decimal inputs are supported.

Reply:
@aditi-pandit, decimal tests for array functions is tracked by this issue: #5940. Would it be fine if the decimal tests for `array_cum_sum` are added after this is resolved?